.travis.yml

```diff
@@ -13,7 +13,7 @@ matrix:
         - git config --global user.name "${GH_NAME}"
         - git config --global user.email "${GH_EMAIL}"
         - echo "machine github.com login ${GH_NAME} password ${GH_TOKEN}" > ~/.netrc
-        - cd docs/site && npm install && GIT_USER="${GH_USER}" npm run publish-gh-pages
+        - cd docs/site && npm install && GIT_USER="${GH_NAME}" npm run publish-gh-pages
```

Comment:
travis log:

```
On branch gh-pages
Your branch is up to date with 'origin/gh-pages'.

nothing to commit, working tree clean
remote: Invalid username or password.
fatal: Authentication failed for 'https://dsngroup@github.com/dsngroup/broke.git/'
Error: Git push failed
```

Reply:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/13215329/49570028-85213700-f970-11e8-9da8-204255924365.png">

It seems that travis doesn't allow an encrypted environmental variable from a pull request (```access token``` in this case). So the travis is unable to git-push here because of the lack of ```access token```. See [pull request and security restrictions](https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions).

Reply:
One way to solve this is to store the access token without encrypting it, but that seems unreasonable.